helpers/view/filter_keyword_input.go

```diff
@@ -0,0 +1,10 @@
+package view
+
+func FilterKeywordPartialInput(query map[string][]string, label string, name string, placeholder string) map[string]interface{} {
```

Comment:
Because `c.Request.URL.Query()` return map[string][]string and from this [comment](https://github.com/gutakk/go-google-scraper/pull/56#discussion_r562531927), yes there will more than one search term.

Reply:
I understand now :-)